Emails/index.js

```diff
@@ -2,11 +2,15 @@ const EmailService = require("../Services/EmailService");
 module.exports = {
   async registrationEmail(data) {
     try {
+      const dev = process.env === "development";
```

Comment:
This line is wrong.

The **process.env** returns an object which if I'm not mistaking has the **NODE_ENV** property to check for environment.
Or better still you could use a package called **dotenv**. Using this package allows you to manage your evn variables in **.env** file.

Please check this out and refactor. Thanks.